dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ExecutorServiceImpl.java

```diff
@@ -893,6 +893,7 @@ protected int createComplementDependentCommand(List<Schedule> schedules, Command
 
         try {
             dependentCommand = (Command) BeanUtils.cloneBean(command);
+            dependentCommand.setId(null);
```

Comment:
I found it's duplicated with #11929. cc @stalary

Reply:
Ok, I removed this change.